jenkins/ci.suse.de/templates/cloud-mkcloud-job-ha-linuxbridge-bonding-template.yaml

```diff
@@ -20,7 +20,6 @@
             TESTHEAD=1
             cloudsource=develcloud{version}
             nodenumber={nodenumber}
-            clusterconfig=data+network+services=2
```

Comment:
change it to =3 ? the default is two  node clusters (otherwise we have to fix the default, which probably makes sense..)

Reply:
done